servers/rendering/rendering_device.h

```diff
@@ -423,6 +423,7 @@ class RenderingDevice : public Object {
 		TEXTURE_USAGE_CAN_COPY_FROM_BIT = (1 << 7),
 		TEXTURE_USAGE_CAN_COPY_TO_BIT = (1 << 8),
 		TEXTURE_USAGE_RESOLVE_ATTACHMENT_BIT = (1 << 9),
+		TEXTURE_USAGE_INPUT_ATTACHMENT_BIT = (1 << 10),
```

Comment:
This and resolve should not be texture modes as far as I know, color attachment and sampling are the right ones for this.

Reply:
I had to add them to make it work. Vulkan validations did not like it when the textures were not properly marked and things did not work.